### DISCUSSION TITLE
Named Entity Recognition Config

### DISCUSSION THREAD

**Discussion by mobab-th** [sepinf-inc/IPED#1549] [General]
Hello,

i think Images and Texts in System Folders can be excluded for NER in the Config File.
Also seems to me the limitation to a maximum file size makes sense.

**Comment by lfcnassif** [comment]
Hi Dennis. We can change to exclude texts and images in system folders by default, seems reasonable. But other image categories I think can bring interesting results if OCR was enabled. A maximum file size to run the module is also a good idea. Thanks.

**Reply by mobab-th** [reply]
Hi Luis,
>But other image categories I think can bring interesting results if OCR was enabled

I agree, therefore the suggestion only limited to system folders.

Keep up the good work, IPED has become a good tool for me in my daily work.
Now it should only be able to read containers from X-Ways Forensics ;-)

**Reply by lfcnassif** [reply]
> Now it should only be able to read containers from X-Ways Forensics ;-)

Unfortunately we don't use/have X-Ways Forensics here, so it would be very hard for us to implement support for it, test containers of new X-Ways versions, provide maintenance and support for them...
